### PR TITLE
Added a couple of FAQ to bug analysis

### PR DESCRIPTION
**Pull request by paoloantinori**

### CODE REVIEW

File: content/faq.md [hunk 1]
```diff
@@ -18,3 +18,23 @@ menu:
 Anyone that wants to integrate services. Syndesis includes a swish UI that enables the user to design integration flows and manage them from their browser.
 
 No coding required... Unless you really want to and then Syndesis allows you to dive into the code, develop your own connectors (if one doesn't already exist), or hack on the integration definition directly.
+
+## How to read logs?
+Logs are currently visible only at infrastructure level, not via the web ui.
+You can get a list of the pods running in Syndesis with the command: `oc get pods -n myproject`.  To read logs from command line, you can then invoke: `oc logs <POD_NAME>`
+
+## How to read the configuration files that are generated by the UI tool
```
Comment (rhuss) [inline content/faq.md]: trailing ? missing

File: content/faq.md [hunk 2]
```diff
@@ -18,3 +18,23 @@ menu:
...
+If you are a developer and you want to check the configuration files corresponding to the Integration that you have designed in the web ui, you can access them with the following commands:
+- `oc cp myproject/<POD_NAME>:/deployments/project-0.1-SNAPSHOT.jar prj.jar`
```
Comment (rhuss) [inline content/faq.md]: You might want to mention that you can get the whole src by copying over `/tmp/src` from the integration pod as this is the directory that is used by the S2I build.

File: content/faq.md [hunk 3]
```diff
@@ -18,3 +18,23 @@ menu:
...
+
+In case your specific error prevents the jvm process (and the pod) to stay running you can still extract files from it.
```
Comment (rhuss) [inline content/faq.md]: Maybe also mention that you might get the previous pods log by `oc logs -p <pod>` ? (well probably not in this section)

File: content/faq.md [hunk 4]
```diff
@@ -18,3 +18,23 @@ menu:
...
+You have to use `docker` to achieve this:
+
+1. attach your `docker` client to Minishift instance: `eval $(minishift docker-env)`
+2. identify the `docker` container corresponding to your integration: `docker ps -a | grep k8s_<NAME_YOU_GAVE_TO_YOUR_INTEGRATION>`
+3. copy out the built archive: `docker cp <CONTAINER_NAME>:/deployments/project-0.1-SNAPSHOT.jar ~/my_integration.jar`
+
+In this file you will find the following configuration files:
+
+- `BOOT-INF/classes/syndesis.yml`
+- `BOOT-INF/classes/mapping-step-*.jso`
```
Comment (rhuss) [inline content/faq.md]: typo: `.jso`

Might be easier to refer to `/tmp/src/resources` which contains these files flat ?

**Comment by paoloantinori** [comment]
Addressed comments + added a different flow to recover data from a dead container.

**Comment by rhuss** [comment]
@kahboom `hugo` in the circle ci build is failing (see the logs of the build). Do you know why and how it could be fixed ?

**Comment by kahboom** [comment]
Taking a look @rhuss , wonder if this has to do with the Hugo version number.

**Pull request merged**